src/domain/index.ts

```diff
@@ -34,3 +34,9 @@ export interface EstimatedWithdrawFee {
   amount: BigNumber;
   USD: number;
 }
+
+export interface EstimatedDepositFee {
+  amount: string;
+  gasPrice: BigNumber | null;
```

Comment:
Taking a look at the code in the thunk we can remove this `null` from here as you are already checking it in the interface.

Reply:
Right! Done.